content/client-server-api/_index.md

```diff
@@ -2042,12 +2036,16 @@ able to accurately aggregate the events.
 #### Relationships API
 
 To retrieve the child events for a parent from the server, the client can call the
-following endpoint with relevant information. This endpoint does not aggregate the child
-events and is instead paginated: clients can perform local aggregation if needed. This
-allows clients to retrieve child events which do not require aggregation but still make
-use of `rel_type`.
+following endpoint. The events returned might require local aggregation by the client,
+however if those same events have bundles of their own then they should be present. A
+hypothetical example would be retrieving all responses to a thread, and those responses
+might have emoji reactions made against them: the client will need to construct the
+thread itself, but the events in the thread should have a reactions bundle on them.
+
+The lack of aggregation behaviour allows clients to retrieve child events which don't
+require aggregation, but do make use of a `rel_type`.
```

Comment:
isn't the definition of a "child event" (in this context) that it "makes use of a `rel_type`"  ?

Reply:
this is trying to clarify the confusion formed around why `/relations` would be listed as an endpoint which does aggregation, but isn't doing the aggregation on the top-level response (instead bundling for the events contained within). 

Very open to suggestions on this.